ActionMenu/UIResources/index.js

```diff
@@ -775,6 +775,9 @@ function load_action_menu(_menu, _settings) {
 	// need an item registry to sync game<>menu update system
 	Object.values(menus).forEach(m => {
 		m.forEach(item => {
+			// initialize the enable property for toggles to their current value
+			if (item.action.toggle)
+				item.enabled = item.enabled ?? item.action.value;
```

Comment:
🤔 doesn't that break other behavior?

action.value is intended to the value of the target when enabled, not if it's enabled. In the code I'm using `default_value` for that but I see it's not used within `build_$item` . That could be the real fix.